src/test/java/org/jenkinsci/plugins/workflow/cps/DSLTest.java

```diff
@@ -420,6 +432,143 @@ public void namedSoleParamForStep() throws Exception {
                 "'org.jenkinsci.plugins.workflow.steps.SleepStep': comment,units", b);
     }
 
+    @Test public void sensitiveVarsLogging() throws Exception {
```

Comment:
```suggestion
    @Issue("JENKINS-63254")
    @Test public void sensitiveVariableInterpolation() throws Exception {
```

Reply:
resolved in 7c8022b5eec0e7ecad65341b3fac3b4c05fbae2b